app/controllers/stocks_controller.rb

```diff
@@ -3,6 +3,9 @@ class StocksController < ApplicationController
   # GET /stocks or /stocks.json
   def index
     @stocks = Stock.all
+    @holdings = Holding.all
+    @user_id = params[:user_id]
+    @holdings = Holding.find_by(id: params[:user_id])
   end
 
   def show
```

Comment:
will probably delete some dito for the next update, since hindi ginagamit sa index page.